scripts/validation/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "validation",
+  "version": "1.0.0",
+  "description": "for validation of embedded examples in asyncapi.md file",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "validate:examples": "node embedded-examples-validation.js"
+  },
+  "author": "Animesh Kumar",
+  "license": "ISC",
+  "devDependencies": {
+    "@asyncapi/parser": "^3.1.0",
+    "js-yaml": "^4.1.0",
+    "json-merge-patch": "^1.0.2",
+    "jsonpath-plus": "^9.0.0",
```

Comment:
You don't need this dependency anymore I believe.

Reply:
Yes, will remove those now!

Reply:
only the jsonpath isn't required

Reply:
achieved thorugh e3d6a78